website/docs/cloud-docs/agents/agents.mdx

```diff
@@ -8,6 +8,19 @@ description: >-
 
 The agent software runs on your own infrastructure. The token you provide when starting the agent assigns it to an HCP Terraform [agent pool](/terraform/cloud-docs/agents/agent-pools).
 
+> #### Quick Start  
+Install and run HCP Terraform agents with the official agent Docker container. 
+`TFC_AGENT_NAME` is the name for your agent. The `TFC_AGENT_TOKEN` field is generated when you create an agent pool.
+`TFC_ADDRESS` is your network address.
+```shell
+export TFC_AGENT_TOKEN=your-token
+export TFC_AGENT_NAME=your-agent-name
+export TFC_ADDRESS=your-address
+docker pull hashicorp/tfc-agent:latest
+docker run -e TFC_AGENT_TOKEN -e TFC_AGENT_NAME -e TFC_ADDRESS hashicorp/tfc-agent
+```
+
+
```

Comment:
for this i don't think a quick start is necessarily super helpful. i think what we want to do here is reorganize and rename some sections. i'm having a hard time showing that using the suggestions tool atm so i'll figure out a different way to do that soon

Reply:
okay here's a commit that shows how we could reorganize this page to have docker image and binary sections https://github.com/hashicorp/terraform-docs-agents/commit/9f57b1f679892bb18913abb018eb09413ce39933

if we reorg something like this then we can link directly to those sections where we need to